cpp/src_prims/cache/cache.h

```diff
@@ -0,0 +1,314 @@
+/*
```

Comment:
General question: This cache prim was merged as part of another PR, no? Why is this still showing up in here?

Reply:
ah.. my bad. I thought cache prim PR #853 was merged already.